source/chapters/1/meteorSnacks.md

```diff
@@ -126,13 +133,16 @@ Meteor.methods({
 })
 ```
 
-Meteor.defer(fn) is same as Meteor.setTimeout(fn, 0), basically it will do a "background" job in a fiber.
+*Meteor.defer(fn)* is same as *Meteor.setTimeout(fn, 0)*. 
+Basically, *Meteor.defer(fn)* will do a "background" job in a fiber (asynchronously...get it ? ).
 
 ## HTTP
 
-Want to use an external REST API ? No problem for Meteor, it has a super simple HTTP package built-in.
-
-http://docs.meteor.com/api/http.html
+"-Would you like an external REST API with that sir ?
+ -Yes of course!
+ -Coming right up!"
+ This is how a conversation between a "hungry" developer and Meteor would look like, if the developer wanted to know if Meteor can serve him an external REST API.
```

Comment:
Maybe we should send him a refference to "Rest API's", maybe he doesn't know about that. 
Immediately after, we could redirect him to that.